src/index.js

```diff
@@ -56,7 +63,12 @@ Divide by 5, then multiply by 9, then add 32
 
 */
 
-
+const filterWeely = weeklyForecast.filter((Forecast)=>{
+  //   return Forecast.length > 5
```

Comment:
This is better naming for the argument passed into the callback of filter, as the same with map it refers to the item of the array, but generally argument names are not in capitalised so maybe `forecast` would be better. 

I can see you have the concept of filter down as you are returning a boolean. 

I think the problem you may have run into is that as Forecast refers to an item in the array which in our case is an object, so it doesn't have a property length. If you console.log(Forecast.length) you will see it logs out the value `undefined` so the expression you are using comes to `return undefined > 5` which always  gives false the value of filterWeekly will be an empty array. 

The property that you want to be looking at is cloudCoverage on the Forecast object

so if `Forecast.cloudCoverage` equals "clear" or if Forecast.cloudCoverage equals "few"

then you want to return true to keep it otherwise return false to discard